sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -1382,7 +1387,7 @@ private Globally(String name, GlobalCombineFn<? super InputT, ?, OutputT> fn,
     public void populateDisplayData(DisplayData.Builder builder) {
       super.populateDisplayData(builder);
 
-      Combine.populateDisplayData(builder, fn, fnClass);
+      Combine.populateDisplayData(builder, fn, fnDisplayData);
```

Comment:
I've added usage comments on [`DisplayData.Builder#include`](https://github.com/apache/incubator-beam/pull/259/commits/c733ae2906ce601d85ffed230bbe110a067fbb74#diff-703d3d7b0209b8b2c08c432d17024809R170) and [`HasDisplayData#populateDisplayData`](https://github.com/apache/incubator-beam/pull/259/commits/c733ae2906ce601d85ffed230bbe110a067fbb74#diff-d4c43f95e94966b9d6a4c0967bc6d70aR57)-- let me know what you think.